Client/Frontend/Strings.swift

```diff
@@ -3605,8 +3605,13 @@ extension String {
         public static let SheetHeaderTitle = MZLocalizedString(
             key: "", // Shopping.Sheet.Title.v118
             tableName: "Shopping",
-            value: "Review quality check",
+            value: "Review checker",
             comment: "Label for the header of the Shopping Experience (Fakespot) sheet")
+        public static let SheetHeaderBetaTitle = MZLocalizedString(
+            key: "", // Shopping.Sheet.Beta.Title.v119
```

Comment:
Curious, we are commenting the key to avoid exporting the string because is not final?